### DISCUSSION TITLE
To make a formulae or not?

### DISCUSSION THREAD

**Discussion by mezantrop** [Homebrew/discussions#4921] [Writing Formulae/Casks] [answered]
### Output of `brew config`

```shell
-
```

### Output of `brew doctor`

```shell
-
```

### Description of issue

A somewhat theoretical question, if you please. To list open sockets, netstat and lsof are used on macs. In BSD world there is a special sockstat tool to make this task easier. This makes me believe, people are trying to seek it on macOS (and in Homebrew) as well. So, recently, I recreated sockstat (https://github.com/mezantrop/sockstat) for macOS utilizing libproc interface. Now, to the point, the question is, should I make and submit a formulae for it, taking in account, potential interest in the future?

**Comment by ZhongRuoyu** [comment, marked as answer]
If you meant to submit the formula to the core tap: formulae there need to meet notability requirements (at least 30 forks, 30 watchers or 75 stars) in order to be accepted. Before your software gains enough recognition, you may want to consider hosting the formula in a [tap](https://docs.brew.sh/Taps) instead. (See also: ["How to Create and Maintain a Tap"](https://docs.brew.sh/How-to-Create-and-Maintain-a-Tap).)

Please note that this has nothing to do with the quality of the software. It's just that there are already thousands of formulae at `homebrew/core`, and we can't unconditionally accept anything. So, we have to draw a line somewhere.

**Reply by mezantrop** [reply]
Ah, ok. Thank you. Well, let's wait for it to gain popularity.
